### PR TITLE
Support methods for object initializers.

### PR DESCRIPTION
**Pull request by zherczeg**
MethodDefinition in ES-2015 12.2.6.

### CODE REVIEW

File: jerry-core/parser/js/js-parser-expr.c [hunk 1]
```diff
@@ -633,6 +633,28 @@ parser_parse_class (parser_context_t *context_p, /**< context */
 } /* parser_parse_class */
 #endif /* !CONFIG_DISABLE_ES2015_CLASS */
 
+#ifndef CONFIG_DISABLE_ES2015_OBJECT_INITIALIZER
+/**
+ * Parse object method.
```
Comment (LaszloLango) [inline jerry-core/parser/js/js-parser-expr.c]: Please add more description, add the related section number as a note to the description.

File: jerry-core/parser/js/js-parser-expr.c [hunk 2]
```diff
@@ -633,6 +633,28 @@ parser_parse_class (parser_context_t *context_p, /**< context */
 } /* parser_parse_class */
 #endif /* !CONFIG_DISABLE_ES2015_CLASS */
 
+#ifndef CONFIG_DISABLE_ES2015_OBJECT_INITIALIZER
```
Comment (LaszloLango) [inline jerry-core/parser/js/js-parser-expr.c]: Is this feature turned on by default in ES5.1 profile?

Comment (zherczeg) [reply jerry-core/parser/js/js-parser-expr.c]: No. It is the opposite.

Comment (rerobika) [reply jerry-core/parser/js/js-parser-expr.c]: Please add a description about the new feature to [the profile documentation](https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/profiles/README.md).

Comment (zherczeg) [reply jerry-core/parser/js/js-parser-expr.c]: The CONFIG_DISABLE_ES2015_OBJECT_INITIALIZER is not a new macro, just a new feature is included into it.

**Pull request merged**